Initial LVGL doc for ESPHome (#3678)
OttoWinter/esphomedocs: components/binary_sensor/lvgl.rst

.. _lvgl-bse:

LVGL Binary Sensor
==================

.. seo::

>     :description: Instructions for setting up a LVGL widget binary sensor.

Reviewer: ```suggestion
    :description: Instructions for setting up an LVGL widget binary sensor.
```

    :image: ../images/lvgl_c_bns.png

> The ``lvgl`` binary sensor platform creates a binary sensor from a LVGL widget

Reviewer: ```suggestion
The ``lvgl`` binary sensor platform creates a binary sensor from an LVGL widget
```

and requires :ref:`LVGL <lvgl-main>` to be configured.

Supported widget is :ref:`lvgl-wgt-btn`. A single binary sensor supports

> a single widget, thus you need to choose among which one's state you want to use.

Reviewer: This reads strangely. I might rephrase:
```suggestion
Supported widget is :ref:`lvgl-wgt-btn`. There is a one-to-one relationship between any given widget
and its associated binary sensor.
```

Author: I find this a bit too "scientific" :) but I do agree it needs improvement.

Reviewer: I also agree -- hence just a suggestion. 😄 Maybe this?:
```suggestion
Supported widget is :ref:`lvgl-wgt-btn`. A single binary sensor supports
only a single widget; it is not possible to have a single binary sensor respond to multiple widgets.
```

Configuration options:
----------------------

- **id** (*Optional*, :ref:`config-id`): Manually specify the ID used for code generation.

> - **name** (**Required**, string): The name of the binary sensor.

Reviewer: These are covered in the "All other options from..." link.
```suggestion
```
